src/storage/SimpleResourceStore.ts

```diff
@@ -128,10 +128,11 @@ export class SimpleResourceStore implements ResourceStore {
    * @returns The resulting Representation.
    */
   private generateRepresentation(data: Quad[], preferences: RepresentationPreferences): Representation {
-    if (preferences.type && preferences.type.some((preference): boolean => preference.value.includes('text/turtle'))) {
-      return this.generateBinaryRepresentation(data);
+    // Always return turtle unless explicitly asked for quads
+    if (preferences.type && preferences.type.some((preference): boolean => preference.value.includes('internal/quads'))) {
```

Comment:
You will like https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining